pages/confirm.js

```diff
@@ -98,8 +100,13 @@ const ResendEmailActions = () => {
   };
 
   return (
-    <Button loading={isLoading} onClick={ask_confirmation_email} type="primary">
-      Pedir Novamente
+    <Button
+      loading={isLoading}
+      onClick={ask_confirmation_email}
+      type="primary"
+      disabled={clicked}
+    >
+      {clicked ? "Pedido enviado" : "Pedir Novamente"}
```

Comment:
On 201, shouldn't we `setClicked(false)` in order to let the user click the button again?

I think a great solution would be using throttle here. Maybe with lodash is the easiest I could find (see [how-and-when-to-debounce-or-throttle-in-react](https://blog.logrocket.com/how-and-when-to-debounce-or-throttle-in-react/)). But there are other react packages for this.

Reply:
Here you get an Ant Design debounce example: https://ant.design/components/select/#components-select-demo-select-users.